gradle/gradle-daemon-jvm.properties

```diff
@@ -0,0 +1,8 @@
+#This file is generated by updateDaemonJvm
```

Comment:
Ah I see. It doesn't actually do dynamic resolution. This is still way better than the status quo though of people running into random issues when using new JDKs.